### ISSUE TITLE
error on sync single file on save with new file in new dir

### ISSUE THREAD

**Issue by yaboroda** [thisboyiscrazy/vscode-rsync#76]
When i create new dir "Model" and create new file "AtiModel.php" in it and save it, extention try to sync it, but failed with error:
`Wed Feb 27 2019 10:55:48 GMT+0300 (MSK) syncing
> rsync -rlptzv --progress --delete --exclude=.git --exclude=.vscode --exclude=.hg --exclude=app/cache /home/username/projects/projectname/sync/vendor/logist/ati-bundle/src/LogistAtiBundle/Model/AtiModel.php username@server.com:/var/www/username/data/www/projectname/vendor/logist/ati-bundle/src/LogistAtiBundle/Model/AtiModel.php 
sending incremental file list
rsync: change_dir#3 "/var/www/username/data/www/projectname/vendor/logist/ati-bundle/src/LogistAtiBundle/Model" failed: No such file or directory (2)
rsync error: errors selecting input/output files, dirs (code 3) at main.c(625) [Receiver=3.0.9]
rsync: connection unexpectedly closed (263 bytes received so far) [sender]
rsync error: error in rsync protocol data stream (code 12) at io.c(226) [sender=3.1.3]`

If i understand correctly it tryes to cd into dir which not created yet

**Comment by thisboyiscrazy** [comment]
Yeah, that would be the case. If you create a directory, then and file on the client side. Then save the file rsync would complain about the directory not existing on the server. I'll check to see if there is an "on directory create" type hook. 

For you can do a "Sync Up" after you create directories, then "onSaveIndijual"  would work.
